subworkflows/local/bam_variant_calling_tumor_only_all/main.nf

```diff
@@ -109,6 +111,13 @@ workflow BAM_VARIANT_CALLING_TUMOR_ONLY_ALL {
         versions = versions.mix(BAM_VARIANT_CALLING_FREEBAYES.out.versions)
     }
 
+    // MSISENSOR
+    if (tools.split(',').contains('msisensorpro') && msisensorpro_baseline) {
```

Comment:
similar here, if the baseline is require than the tool shouldn't run on an empty channel

Reply:
I remember I tried without it but it failed for some reason that I can't remember now so I think it was necessary.